includes/admin/settings/class-wcmp-settings-data.php

```diff
@@ -805,9 +805,9 @@ private function get_section_export_defaults_main()
             ],
             [
                 "name"      => WCMYPA_Settings::SETTING_EMPTY_PARCEL_WEIGHT,
-                "label"     => __("Empty parcel weight (grams)", "woocommerce-myparcel"),
+                "label"     => __("Empty parcel weight", "woocommerce-myparcel"),
                 "help_text" => __(
-                    "Default weight of your empty parcel, rounded to grams.",
+                    "Default weight of your empty parcel.",
```

Comment:
Weet de gebruiker nu wel of het in kg/gram moet?

Reply:
Hangt van de instellingen af van woocommerce. Wanneer je kg hebt bijv dan ook hier kg